TDDPOC/TDDPOC/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Mvc;
+using TDDPOC.Service;
+
+namespace TDDPOC.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private readonly IWeatherForecastService _weatherForecastService;
+
+        public WeatherForecastController(IWeatherForecastService weatherForecastService)
+        {
+            _weatherForecastService = weatherForecastService;
+        }
+
+        [HttpGet("{summary}")]
+        [ProducesResponseType(typeof(WeatherForecast), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetBySummary(string summary)
```

Comment:
service method is not async and we are not using await keyword anywhere then why parent method is async

Reply:
removed the async task from controller